src/sidebar/components/Annotation/AnnotationShareControl.js

```diff
@@ -37,6 +39,23 @@ function selectionOverflowsInputElement() {
   return isIOS();
 }
 
+/**
+ *
+ * @param {object} props
+ *   @param {string} [props.classes] - Optional additional CSS classes
+ */
```

Comment:
I probably would have shortened this to just:

```js
/** @param {{ classes: string }} */
```

Since the meaning of the parameter is pretty obvious from its name.